.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,35 +9,44 @@
   NOTE: use `feat`, `fix` and `perf` for user facing changes that will be part of
   release notes.
 -->
+
 ## Description
+
 <!--- Describe your changes in detail -->
 <!--- If applicable, describe (or illustrate) architecture flow -->
 <!--- If the UI changes in a non-trivial way, consider adding screenshots/video illustrating the new flows -->
 
 ### Checklist
+
 - [ ] New tests and/or benchmarks are included
 - [ ] Documentation is changed or added
 - [ ] If this change updates the UI, screenshots/videos are added and a design review is requested
 - [ ] If this change could impact the load on the MongoDB cluster, please describe the expected and worst case impact
 - [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)
 
 ## Motivation and Context
+
 <!--- Why is this change required? What problem does it solve? -->
 <!--- If it's updating a dependency, link to the Pull Request that originally introduced the fix -->
+
 - [ ] Bugfix
 - [ ] New feature
 - [ ] Dependency update
 - [ ] Misc
 
 ## Open Questions
+
 <!--- Any particular areas you'd like reviewers to pay attention to? -->
 
 ## Dependents
+
 <!--- If applicable, link PRs/commits that this PR is dependent on or is a dependency of. -->
 
 ## Types of changes
+
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] *Backport Needed*
+
+- [ ] _Backport Needed_
```

Comment:
These changes started to show up when I merged the main in my branch. I think these are due to precommit hook, where prettier reformatted the file.